components/01-atoms/text/text/07-byline.twig

```diff
@@ -0,0 +1,35 @@
+{#
+/**
+ * Available variables:
+ * - byline_base_class - the base classname
+ * - byline_modifiers - array of modifiers to add to the base classname
+ * - byline_blockname - blockname prepended to the base classname
+ * - byline_content - the content of the byline (typically text)
+ * - byline_author - the author of the byline (typically text)
+ * - byline_pre - prefaces the author of the byline (typically text)
+ *
+ * Available blocks:
+ * - byline_content - used to replace the content of the blockquote with something other than text
+ *   for example: A formatted text field in Drupal
+ */
+#}
+
+{% set byline__base_class = byline__base_class|default('byline') %}
+
+{# {% set date_time__attributes = {
+  datetime: date_time__start|date(format__iso),
+  class: bem('date-time'),
+} %} #}
+
+
+
+  <byline {{ add_attributes(byline__attributes) }}>
+    {% block byline_content %}
+      {{ byline_content }}
```

Comment:
This feedback also applies to the comments above: We should be using double underscores to separate different parts of a component and single underscores when it's suppose to act as a space within a name of a variable. So the naming convention should be something like `[component_name]__[component_part]`. 

Example `{{ byline__author }} indicates ownership of this variable to the byline component and `author` is the specific value we want to display. This could also be extrapolated to `{{ byline__first_author }}` and `{{ byline__second_author }}` <-- this example follows the same naming convention but shows that `first_author` and `second_author` are specific values and not separate. Taking this naming convention even further `{{ byline__first_author__image }}` This variable name indicates this is a byline specific variable, a first_author element/value is expected, and `image` indicates we want to print an image.
:point_up: hope I explained the reasoning behind the naming convention but let me know if you want to chat further.